cppSettings.gradle

```diff
@@ -180,7 +180,7 @@ subprojects {
                         // Gradle auto-adds the -m32 argument to the linker and compiler. Our compiler only supports
                         // arm, and doesn't understand this flag, so it is removed from both
                         cppCompiler.withArguments { args ->
-                            args << '-std=c++1y' << '-Wformat=2' << '-Wall' << '-Wextra' << '-Werror' << '-pedantic'
+                            args << '-std=c++11' << '-Wformat=2' << '-Wall' << '-Wextra' << '-Werror' << '-pedantic'
```

Comment:
I'm thinking if the user plugins with -std=c++1y, we should do the same, just to make sure nothing breaks unexpectedly.  Let's not make this change in this PR.